source/dem/pw_linear_force.cc

```diff
@@ -106,8 +116,12 @@ PWLinearForce<dim>::calculate_pw_contact_force(
     &           pw_pairs_in_contact,
   const double &dt,
   std::unordered_map<types::particle_index, Tensor<1, dim>> &momentum,
-  std::unordered_map<types::particle_index, Tensor<1, dim>> &force)
+  std::unordered_map<types::particle_index, Tensor<1, dim>> &force,
+  const DEMSolverParameters<dim> &                           dem_parameters)
 {
+  PWContactForce<dim>::update_boundary_velocity(dem_parameters);
+  PWContactForce<dim>::ForceOnWall.clear();
+  PWContactForce<dim>::TorqueOnWall.clear();
```

Comment:
Should they be cleared or initialized to zero? I would think that they should be initialized to zero.

Reply:
Thought that clear initialized to zero. Is it better?